### PR TITLE
fix: Allow localhost

### PR DESCRIPTION
**Pull request by JamesCullum**
## Description of the change
The whitelist does not work if the container is in the host network, as postfix does not allow sending from localhost. This simple change allows localhost to send emails as well.

## Motivation and Context
Using the host network does not work.

## How Has This Been Tested?
Tested the change locally successful

## Types of Changes
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation (adding or updating documentation)

## Checklist:
- [ ] My change requires a change to the documentation and I have updated the documentation accordingly.
- [ ] My change adds a new configuration variable and I have updated the `.env.example` file accordingly.

And lastly, many thanks for taking your time to help us improve this project !

### CODE REVIEW

**Comment by JamesCullum** [comment]
Hey @juanluisbaptiste the issue is the list of allowed senders - at the moment this only allows IPs in the range of 10.0.0.0/8, 172.16.0.0/12, 192.168.0.0/16 to send emails.

However, if I use the host network (meaning that all of them consume the host ports), this means I need to figure out the IP of the container and communicate directly with it. However the reason I use the host network is so that I don't need to do that, and instead just use port 25 on localhost. However, because 127.0.0.1 & localhost is not allowed, the SMTP will reach the backend but will be restricted by the IP limitation.

So basically this fix allows to use the host network properly and just use localhost as destination for any SMTP tool, instead of having to figure out the containers internal, temporary IP.
